### PR TITLE
Gabby/Edit javadocs and user guide

### PR DESCRIPTION
**Pull request by GabriellaTeh**
Edited examples in user guide and javadocs. 
Need to change more javadocs after the instances of "address book" is changed.

### CODE REVIEW

File: src/main/java/seedu/address/MainApp.java [hunk 1]
```diff
@@ -48,7 +48,7 @@
 
     @Override
     public void init() throws Exception {
-        logger.info("=============================[ Initializing PlaNus ]===========================");
+        logger.info("=============================[ Initializing PlaNUS ]===========================");
```
Comment (BobbyZhouZijian) [inline src/main/java/seedu/address/MainApp.java]: I think we are changing all these to PlaNus

**Review by BobbyZhouZijian — COMMENTED** [review]
Rest lgtm. Just wonder the PlaNus part.

File: src/main/java/seedu/address/model/task/Description.java [hunk 2]
```diff
@@ -16,7 +16,7 @@
             + "example => desc:Tell Zijian to review my PR!!! Repeat, this is urgent!!! You get it?";
 
     /*
-     * The first character of the description must not be a whitespace,
+     * The first character of the address must not be a whitespace,
```
Comment (MarcTzh) [inline src/main/java/seedu/address/model/task/Description.java]: why did you change this?

File: src/main/java/seedu/address/model/task/State.java [hunk 3]
```diff
@@ -1,5 +1,9 @@
 package seedu.address.model.task;
 
+/**
+ * Represents a Task's state.
+ * A task's state can be either done or pending.
```
Comment (MarcTzh) [inline src/main/java/seedu/address/model/task/State.java]: can change to complete or incomplete

File: src/main/java/seedu/address/model/task/Status.java [hunk 4]
```diff
@@ -24,14 +24,14 @@ public Status(State state) {
     }
 
     /**
-     * Constructs a default {@code Status} that have a state of pending.
+     * Constructs a default {@code Status} that has a pending state.
```
Comment (MarcTzh) [inline src/main/java/seedu/address/model/task/Status.java]: has a state of incomplete

File: src/main/java/seedu/address/model/task/Status.java [hunk 5]
```diff
@@ -24,14 +24,14 @@ public Status(State state) {
...
      */
     private Status() {
         value = State.PENDING;
     }
 
     /**
-     * Returns a default Status object which have a pending state
+     * Returns a default Status object which has a pending state
```
Comment (MarcTzh) [inline src/main/java/seedu/address/model/task/Status.java]: state of incomplete

**Review by MarcTzh — APPROVED** [review]
LGTM
